module/VuFind/src/VuFind/View/Helper/Root/EscapeOrCleanHtml.php

```diff
@@ -91,7 +92,7 @@ public function __invoke(
             }
             $value = (string)$value;
         }
-        return $this->escape($value);
+        return $this->escape($value ?? '');
```

Comment:
This _seems_ reasonable -- if you've invoked it with other non-null params, you presumably do want some string back, so if value is null then I guess empty string is best.  But the alternative is to return $this and as above I'm not sure what use case that serves.

Reply:
Looks like the root cause was actually a record driver inappropriately returning null -- so see #4322 for a different solution to the problem. I still think it's worth revising this PR, but if we can merge that other one, we can fix the problem right away.

Reply:
Note that this PR no longer fixes the broken test -- we need #4322 for that. But it will work once that is merged.